cmd/integration-tests/integration-tests.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	aggregdepscore "github.com/DataDog/aggregated-dependency-score"
+)
+
+func main() {
+	err := run()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "ERROR: "+err.Error())
+		os.Exit(1)
+	}
+}
+
+func run() (err error) {
+	testData := struct {
+		RealCases []testCase `json:"real_cases"`
+	}{}
+
+	testDataBytes, err := os.ReadFile("test-data.json")
+	if err != nil {
+		return fmt.Errorf("reading test data: %w", err)
+	}
```

Comment:
[nit] why not using go embed instead?

Reply:
doesn't go embedding require that the file is "in the package"?

Here it's outside so that it can be reused by other implementations.

Also later this will let us use different data files if we want.